presto-iceberg/src/main/java/com/facebook/presto/iceberg/statistics/StatisticsFileCache.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg.statistics;
+
+import com.facebook.presto.spi.statistics.ColumnStatistics;
+import com.google.common.cache.Cache;
+import com.google.common.cache.ForwardingCache.SimpleForwardingCache;
+
+public class StatisticsFileCache
+        extends SimpleForwardingCache<StatisticsFileCacheKey, ColumnStatistics>
+{
```

Comment:
Can you add cache metrics ? We do this e.g in [CachingDirectoryLister](https://github.com/prestodb/presto/blob/e2ec28f6ebca39357bed856441f66aebce9b2667/presto-hive/src/main/java/com/facebook/presto/hive/CachingDirectoryLister.java#L159-L205).

Reply:
We already expose cache metrics? It's done here: https://github.com/prestodb/presto/blob/961f4beb44d853ae51b3152b72d4b4496f25857f/presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergCommonModule.java#L183-L184